packages/v4/src/content/accessibility/tabs/tabs.md

```diff
@@ -0,0 +1,27 @@
+---
+id: Tabs
+section: components
+---
+
+**Tabs** allow users to navigate between views within the same page or context.
+
+**Keyboard users** should be able to use **Tab** and **Shift + Tab** navigate to and between tabs.
+Keyboard users should also be able to select tab using **Enter** and **Space**. Disabled tabs cannot receive
+browser focus. Whenever a disabled tab needs to be able to receive browser focus, such as in the case of
+disabled tabs with a tooltip, set `aria-disabled` to true.
```

Comment:
Minor suggested tweaks here.

```suggestion
Keyboard users should also be able to select a tab using **Enter** and **Space**. Disabled tabs cannot receive
browser focus by default. Whenever a disabled tab needs to be able to receive browser focus, such as in the case of
disabled tabs with a tooltip, use the `isAriaDisabled` prop instead of `isDisabled`.
```